src/AutoRest.CSharp.V3/CodeGen/CsProjWriter.cs

```diff
@@ -17,7 +17,7 @@ public bool WriteCsProj(Configuration configuration)
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputPath>bin</OutputPath>
     <PublishDir>$(OutputPath)</PublishDir>
-    <NoWarn>SA1649</NoWarn>
+    <NoWarn>SA1649,CS8618</NoWarn>
```

Comment:
Since we currently do not have nullable references removed, this is a workaround to allow projects to not error about non-null properties in models. I attempted, at first, to make all of them nullable, but that causes more issues because the caller complains with another warning about "This value may be null". Instead, ignoring this warning for now is appropriate, until we remove nullable references for the development of the project.

Reply:
Set nullable project property to annotation: https://github.com/Azure/autorest.csharp/blob/feature/v3/test/AutoRest.TestServer.Tests/AutoRest.TestServer.Tests.csproj#L6

Reply:
Done.

Reply:
With annotations you don't need the CS8618 skipped anymore.